src/TelegramFile.php

```diff
@@ -63,7 +63,7 @@ public function file(mixed $file, string $type, string $filename = null): self
             'contents' => is_resource($file) ? $file : fopen($file, 'rb'),
         ];
 
-        if (null !== $filename) {
+        if ($filename !== null) {
```

Comment:
Why is this being swapped here and other relevant changes?

Reply:
I don't know. GitHub action bot did that in a commit named style fix

Reply:
Must be pint's rule.